dive_core/generatePm4Info_adreno.py

```diff
@@ -823,6 +825,11 @@ def outputPacketInfo(pm4_info_file, registers_et_root, enum_index_dict, opcode_d
       # Keep track of instance #. Only the 1st instance belongs in the vector. The rest are in the multimap.
       if opcode not in packet_type_instances:
         packet_type_instances[opcode] = 1
+        if (opcode == 0):
+          pm4_info_file.write("    // For descriptors, we purposefully try to include them as \"packets\" for easier parsing.\n")
+          pm4_info_file.write("    // They're not technically PM4 packets, hence the 0x0.\n")
+          pm4_info_file.write("    // It's used: \"const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);\".\n")
```

Comment:
PEP 8 encourages using single quotes here to avoid backslashes (https://peps.python.org/pep-0008/#string-quotes). You've got some options:

Either--given that this includes code samples--you could use backticks instead (like Markdown).

```suggestion
          pm4_info_file.write("    // It's used: `const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);`.\n")
```

Or: keep the code in double quotes and replace the contractions (to avoid backslashes on the single quote):

```suggestion
          pm4_info_file.write('    // It is used: "const PacketInfo *packet_info_ptr = GetPacketInfo(0, sharp_struct_name);".\n')
```

Reply:
Thanks for the suggestion, fixed